demisto_sdk/dev_tools/runner.py

```diff
@@ -0,0 +1,135 @@
+
+        log_ids = []
+
+        for entry in ans:
+            # ans should have entries with `contents` - the readable output of the command
+            if entry.parent_content:
+                print_color('### Command:', LOG_COLORS.YELLOW)
+                print(entry.parent_content)
+            if entry.contents:
+                print_color('## Readable Output', LOG_COLORS.YELLOW)
+                if entry.type == ERROR_ENTRY_TYPE:
+                    print_error(entry.contents + '\n')
+                else:
+                    print(entry.contents + '\n')
+
+            # and entries with `file_id`s defined, that is the fileID of the debug log file
+            if entry.type == DEBUG_FILE_ENTRY_TYPE:
+                log_ids.append(entry.id)
+
+        return log_ids
+
+    def _export_debug_log(self, log_ids: list):
+        """Retrieve & rexport debug mode log files
+
+        Args:
+            log_ids (list): artifact ids of the log files
+        """
+        if self.debug_mode == '-':
+            print_color('## Detailed Log', LOG_COLORS.YELLOW)
+            for log_id in log_ids:
+                result = self.client.download_file(log_id)
+                with open(result, 'r+') as log_info:
+                    for line in log_info:
+                        if self.SECTIONS_HEADER_REGEX.match(line):
+                            print_color(line, LOG_COLORS.YELLOW)
+                        elif self.FULL_LOG_REGEX.match(line):
+                            print_color('Full Integration Log:', LOG_COLORS.YELLOW)
+                        else:
+                            print(line)
+        else:
+            with open(self.debug_mode, 'w+b') as fout:
+                for log_id in log_ids:
+                    result = self.client.download_file(log_id)
+                    with open(result, 'r+') as log_info:
+                        for line in log_info:
+                            fout.write(line.encode('utf-8'))
+            print_color(f'Debug Log successfully exported to {self.debug_mode}', LOG_COLORS.GREEN)
+
+    @staticmethod
+    def add_sub_parser(subparsers):
+        from argparse import ArgumentDefaultsHelpFormatter
+        description = f"""Run integration command on remote Demisto instance in the playground.
+        {Client.DEMISTO_API_KEY_ENV} environment variable should contain a valid Demisto API Key."""
+        parser = subparsers.add_parser('run', help=description, formatter_class=ArgumentDefaultsHelpFormatter)
+        parser.add_argument("-q", "--query", help="The query to run", required=True)
+        parser.add_argument("-u", "--url", help="Base URL of the Demisto instance", required=True)
+        parser.add_argument("-k", "--insecure", help="Skip certificate validation", action="store_true")
+        parser.add_argument("-v", "--verbose", help="Verbose output", action='store_true')
+        parser.add_argument(
+            "-D", "--debug-mode", metavar="DEBUG_LOG",
```

Comment:
lets update the name to debug instead of debug-mode as it is a bit confusing now, here, and in the code itself please